### PR TITLE
ADD keysize related limitation in documentation

### PR DESCRIPTION
**Pull request by fgalan**
Documentation pieces related with issue https://github.com/telefonicaid/fiware-orion/issues/1289

### CODE REVIEW

File: doc/manuals/user/known_limitations.md [hunk 1]
```diff
@@ -66,3 +66,8 @@ constrains applies to the one that Orion supports in the NGSIv1 API (NGSIv2 does
 * P[n]Y[n]M[n]DT[n]H[n]M[n]S is supported (P[n]W format is not supported)
 * Decimal fractions are supported for seconds (S), but not in other elements.
 
+## Entity fields size limitation
+
+Due to underlying DB limitations (see details [here](https://github.com/telefonicaid/fiware-orion/issues/1289)),
+the combined size (sum) of entity id, entity type and entity service path cannot exceed 1014 characters. If you
```
Comment (kzangeli) [inline doc/manuals/user/known_limitations.md]: I read the 1014 and I thought to myself ... that's a typo!
Then I remembered, 10 bytes of overhead somewhere, I looked it up myself the other day! :-)
However, practically everybody will assume it's a typo, unless a short explanation of those 'lost 10 bytes' is given.

Comment (fgalan) [inline doc/manuals/user/known_limitations.md]: Fixed in commit d90a46f34b9386798f0d54686c98ff73293452be

File: doc/manuals/deprecated.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ The following table provides information about the last Orion version supporting
 
 | **Removed feature**                                                        | **Last Orion version supporting feature** | **That version release date**   |
 |----------------------------------------------------------------------------|-------------------------------------------|---------------------------------|
-| XML API                                                                    | Not yet removed (planned: 0.28.0)         | February 29th, 2016             |
+| XML API                                                                    | 0.28.0                                    | February 29th, 2016             |
```
Comment (fgalan) [inline doc/manuals/deprecated.md]: Not actually related with this PR, but a small modification that was pending...

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**